booking/src/main/java/org/dcsa/conformance/standards/booking/model/PersistableCarrierBooking.java

```diff
@@ -295,21 +288,16 @@ public void putBooking(String bookingReference, ObjectNode newBookingData) {
     } else {
       setBooking(newBookingData);
     }
-    var bookingState = getOriginalBookingState();
-    if(!(PENDING_UPDATE.equals(bookingState) || PENDING_AMENDMENT.equals(bookingState) )) {
-      removeFeedbacks();
-    }
+
   }
 
   private void ensureFeedbacksExist(ObjectNode booking) {
     booking
-      .putArray("feedbacks")
-      .addObject()
-      .put("severity", "WARN")
-      .put("code", "PROPERTY_VALUE_HAS_BEEN_CHANGED")
-      .put(
-        "message",
-        "Please perform the changes requested by the Conformance orchestrator");
+        .putArray(FEEDBACKS)
+        .addObject()
+        .put("severity", "WARN")
+        .put("code", "PROPERTY_VALUE_HAS_BEEN_CHANGED")
+        .put("message", "Please perform the changes requested by the Conformance orchestrator");
```

Comment:
```suggestion
        .put("severity", "ERROR")
        .put("code", "PROPERTY_VALUE_MUST_CHANGE")
        .put("message", "Please change VesselName to 'King of the Seas'")
        .put("jsonPath", "$.vessel.name")
        .put("property", "name");
```

Reply:
@palatsangeetha would this work? You can afterwards test **if** `vesselName` has been changed in a UC3